Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Build
+# docker build . -t cyclonedx/cyclonedx-node:test
+
+# Usage
+# docker run --rm \
+#   -v `pwd`:/src \
+#   -w /src \
+#   cyclonedx/cyclonedx-node:test -o /src/bom.xml /src
+
+FROM node:16.13-alpine3.12
+
+WORKDIR /usr/src/cyclonedx-bom
+
+COPY package*.json /usr/src/cyclonedx-bom/
+RUN npm ci --only=production
+
+COPY . /usr/src/cyclonedx-bom
+
+ENTRYPOINT ["/usr/src/cyclonedx-bom/bin/cyclonedx-bom"]
+CMD ["-h"]
```

Comment:
Yes, right now if you do not add any arguments it will just show  the help.

In my opinion is a good practice to avoid unexpected behavior and it is the same logic used by [cyclonedx-dotnet](https://github.com/CycloneDX/cyclonedx-dotnet/blob/master/Dockerfile).

But also in this case it is not a strong opinion, as you prefer.